SmartDeviceLink/private/SDLMenuManager.m

```diff
@@ -552,7 +552,34 @@ - (BOOL)sdl_isDynamicMenuUpdateActive:(SDLDynamicMenuUpdatesMode)dynamicMenuUpda
 }
 
 - (BOOL)sdl_artworkNeedsUpload:(SDLArtwork *)artwork {
```

Comment:
I think this method should probably be added to the `SDLFileManager` as a public method and removed from all these classes. Could you discuss that with Bilal?

Reply:
Sure!